### ISSUE TITLE
Data should return its values instead of ID's.

### ISSUE THREAD

**Issue by DevNebulae** [DeltaSAM/api#21]

**Comment by cascer1** [comment]
I'm not sure what you mean by this. Please add an example with both the current response and what you would like as response.

**Comment by DevNebulae** [comment]
```
"id": 1,
        "projectID": 1,
        "authorID": 2462532,
        "date": "2015-12-01T23:00:00.000Z",
        "grade": 6,
        "remark": "We're doing kinda well I guess"
      }
```

Should be

```
"id": 1,
        "project": {
            "id": 2,
            "name": "LBS",
            "startDate": "2015-08-31T22:00:00.000Z",
            "endDate": null,
           "members": ...
        },
        "authorID": {
            "userID": 2462532,
            "firstName": "Cas",
            "lastName": "Eliëns",
            "email": "cas@eliens.co"
        },
        "date": "2015-12-01T23:00:00.000Z",
        "grade": 6,
        "remark": "We're doing kinda well I guess"
      }
```

**Comment by cascer1** [comment]
Closed in 687319cc7093b7cc8c9bacf3c100ac7deb0ff661
